ci/jobs/projects.yaml

```diff
@@ -0,0 +1,110 @@
+- project:
+    name: build-automation
+    jobs:
+     - 'build-automation-repo-{release_config}'
+    release_config:
+     - 2.6-dev
+     - 2.6-testing
+     - 2.6-release
+     - 2.7-dev
+     - 2.8-dev
+     - master
+
+- project:
```

Comment:
I agree, but think it's worthwhile to do it this way in the scope of this PR. The goal with the projects file in this PR was simply "Make sure the projects file points at every job file", and not "Reorganize the job files". That comes next. :)